src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -128,7 +128,7 @@ internal ReadOnlySpan(ref T ptr, int length)
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public ref T DangerousGetPinnableReference()
+        internal ref readonly T DangerousGetPinnableReference()
         {
             return ref _pointer.Value;
```

Comment:
We will also want to change this to return null for empty Spans, and maybe rename it to GetPinnableReference. It can wait for future change that makes it public again.